src/components/v2/Select/index.tsx

```diff
@@ -0,0 +1,90 @@
+/** @jsxImportSource @emotion/react */
+import React, { useState } from 'react';
+import MenuItem from '@mui/material/MenuItem';
+import FormControl from '@mui/material/FormControl';
+import { SelectChangeEvent } from '@mui/material/Select';
+import { Select as MuiSelect } from '@mui/material';
+import Typography from '@mui/material/Typography';
+
+import { Icon } from '../Icon';
+import { TextButton } from '../Button';
+import { SELECTED_MENU_ITEM_CLASSNAME, useStyles } from './styles';
+
+type Option = {
+  value: string;
+  label: string;
+};
+
+interface ISelectProps {
+  className?: string;
+  options: Option[];
+}
+
+const DEFAULT_OPTION = {
+  value: '',
+  label: 'All',
+};
+
+export const Select = ({ className, options }: ISelectProps) => {
+  const [selectedValue, setSelectedValue] = React.useState(DEFAULT_OPTION.value);
```

Comment:
The state and its modifier function should be passed through props, as we'll always need to use the state from the parent component that renders it